dashboard/db/schema.rb

```diff
@@ -38,12 +38,13 @@
   end
 
   create_table "ap_school_codes", id: false, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+    t.integer  "school_year"
     t.string   "school_code", limit: 6,  null: false
     t.string   "school_id",   limit: 12, null: false
     t.datetime "created_at",             null: false
     t.datetime "updated_at",             null: false
-    t.index ["school_code"], name: "index_ap_school_codes_on_school_code", unique: true, using: :btree
-    t.index ["school_id"], name: "index_ap_school_codes_on_school_id", unique: true, using: :btree
+    t.index ["school_code", "school_year"], name: "index_ap_school_codes_on_school_code_and_school_year", unique: true, using: :btree
+    t.index ["school_id"], name: "fk_rails_08d2269647", using: :btree
```

Comment:
was this name automatically generated? I've never seen something like that before.

Reply:
Yeah I was a little confused by this as well, I searched schema.rb and for `fk_rails` and found 10 instances of indices that look like this. So I think it's ok?

Reply:
If it’s not too much burden, it’s worth giving the foreign key a friendly name.

Reply:
It looks like there's a distinction between indices and foreign keys -- all the foreign keys in our DB have names like `fk_rails...`, and all indices have nice readable names like what you posted here. I think that's just a result of the default (Rails docs says `add_index` names it a nice name by default, and `add_foreign_key` adds a `fk_rails...` type name. Better to go with convention (which has the ugly name) or something more readable?

Reply:
Good point.  The Rails convention is fine.